roles/openshift_aws/filter_plugins/openshift_aws_filters.py

```diff
@@ -9,17 +9,17 @@ class FilterModule(object):
     ''' Custom ansible filters for use by openshift_aws role'''
 
     @staticmethod
-    def build_instance_tags(clusterid, status='owned'):
+    def build_instance_tags(clusterid):
```

Comment:
Is this filter strictly necessary?  I'm a fan of doing this type of thing in ansible directly, when possible.

Reply:
If we can easily create a dictionary with
```
 "kubernetes.io/cluster/{{ clusterid }}": "{{ clusterid }}"
```

Ansible does not allow creation of dictionary keys to be variables.  For this reason I chose a filter_plugin.  I'm open for suggestions.